### ISSUE TITLE
MUC Chatroom does not revceive messages

### ISSUE THREAD

**Issue by francescomilizia** [jsxc/jsxc#271]
Hello,
I've problems with chatrooms.
The first time I join to a chatroom, it works fine. But if I close the chatroom panel (clicking on the X icon) or if I explicitly leave the room, the next time I open / join it I'm not able to receive messages. Reloading the client and reconnecting to the server does not help.
Other participants can see my messages and are notified when I enter or exit the room.
I've tried with different room configurations and with several XMPP servers, but the issue was the same.

Can you help me?

Thank you.

**Comment by sualko** [comment]
Can you test our latest beta build from the release section? Maybe it is already fixed.

**Comment by francescomilizia** [comment]
I've tried the last beta build, but I've the same problems, if not more. This time I was not even able to send messages to the room.

**Comment by sualko** [comment]
Thanks for testing. I will deal with that.

**Comment by sualko** [comment]
I could reproduce your issue and will fix it asap.

>  if not more

I am sorry, did not update muc to use our latest api.

**Comment by francescomilizia** [comment]
Thank you, I appreciate your commitment.

**Comment by francescomilizia** [comment]
Hello,
I've tried your last beta release, but I still cannot receive chatroom messages. I've also noted that the chatroom menu in the contact list is not working.

Thank you.

**Comment by sualko** [comment]
Sorry, I can not confirm your issue. Did you a hard reload? Maybe it helps if you clear your cache.

Which chatroom menu do you mean?

**Comment by francescomilizia** [comment]
I've started from scratch. I checked out the master branch from github, mounted your example dir under apache using a new URL, and changed just the server url inside "example.js" to point my openfire 4.0.1 installation.
I can see incoming messages on the private chat, but the MUC still does not work. 

However the menu issue is solved.

Can I try something else or provide you other informations?
